f2/src/utils/checkIfAvailable.js

```diff
@@ -3,10 +3,9 @@ require('dotenv').config()
 const submissionsPerDay = process.env.SUBMISSIONS_PER_DAY
 const secondsBetweenRequests = process.env.SECONDS_BETWEEN_REQUESTS
 
-if (!submissionsPerDay || !secondsBetweenRequests)
-  console.error('ERROR: Availability not configured')
-else console.info('Availability configured')
-
+if (!submissionsPerDay || !secondsBetweenRequests) {
+  console.error('ERROR: SUBMISSIONS_PER_DAY or SECONDS_BETWEEN_REQUESTS not configured.  The Server will constantly report unavailable as a result.')
```

Comment:
If **availableData.lastRequested** is not set **OR** if there has not been a request today the service will return **true**. Only if these checks evaluate **false** will the service check **submissionsPerDay** or **secondsBetweenRequests**. We should change the message or rearrange the logic to be consistent.